src/Kunstmaan/UtilitiesBundle/DependencyInjection/KunstmaanUtilitiesExtension.php

```diff
@@ -20,7 +20,13 @@ class KunstmaanUtilitiesExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $this->processConfiguration($configuration, $configs);
+        $config = $this->processConfiguration($configuration, $configs);
+
+        if (isset($config['cipher'], $config['cipher']['secret'])) {
+            $container->setParameter('kunstmaan_utilities.cipher.secret', $config['cipher']['secret']);
+        } else if (!$container->hasParameter('kunstmaan_utilities.cipher.secret')) {
```

Comment:
Remove this check since default value has already been set.

Reply:
I have removed the default value in Configuration.php